src/TensorFlowNET.Core/Python.cs

```diff
@@ -309,6 +309,36 @@ public static __object__ getattr(object obj, string key, params Type[] ___parame
             }
             return (__object__)((object[] args) => { return "NaN"; });
         }
+        public static IEnumerable TupleToEnumerable(object tuple)
```

Comment:
We are targeting netstandard2.0, you this shouldn't be done via reflection. It should be done by `System.Runtime.CompilerServices.ITuple`.